digdag-core/src/main/java/io/digdag/core/agent/ConfigEvalEngine.java

```diff
@@ -1,44 +1,38 @@
 package io.digdag.core.agent;
 
-import java.util.List;
-import java.util.Map;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.charset.Charset;
-import javax.script.ScriptEngine;
-import javax.script.ScriptEngineManager;
-import javax.script.ScriptException;
-import javax.script.Invocable;
-import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.annotations.VisibleForTesting;
 import com.google.common.collect.ImmutableList;
 import com.google.common.io.CharStreams;
 import com.google.inject.Inject;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import jdk.nashorn.api.scripting.NashornScriptEngineFactory;
 import io.digdag.client.config.Config;
+import io.digdag.client.config.ConfigException;
 import io.digdag.spi.TemplateEngine;
 import io.digdag.spi.TemplateException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import static java.nio.charset.StandardCharsets.UTF_8;
 
 public class ConfigEvalEngine
     implements TemplateEngine
 {
     private static Logger logger = LoggerFactory.getLogger(ConfigEvalEngine.class);
 
-    private static final String DIGDAG_JS_RESOURCE_PATH = "/io/digdag/core/agent/digdag.js";
-
-    private static final String[] LIBRARY_JS_RESOURCE_PATHS = {
-        "/io/digdag/core/agent/moment.min.js"
+    // array of (name, resourceName)
+    private static final String[][] LIBRARY_JS_RESOURCES = {
+        new String[] { "digdag.js", "/io/digdag/core/agent/digdag.js", "digdag.js" },
+        new String[] { "moment.min.js", "/io/digdag/core/agent/moment.min.js", "moment.min.js" },
```

Comment:
Comment says each row has two column (name, resource), but there are 3 columns.
3rd column is used ?